src/localize/languages/pt-pt.json

```diff
@@ -0,0 +1,467 @@
+{
+  "common": {
+    "frigate_card": "Cartão Frigate",
+    "frigate_card_description": "Um cartão da Lovelace para usar com Frigate",
+    "live": "Ao Vivo",
+    "no_media": "sem média",
+    "recordings": "Gravações",
+    "version": "Versão"
+  },
+  "config": {
+    "cameras": {
+      "camera_entity": "Entidade da Câmera",
+      "dependencies": {
+        "all_cameras": "Mostrar eventos para todas as câmeras nesta câmera",
+        "cameras": "Mostrar eventos para câmeras específicas nesta câmera",
+        "editor_label": "Opções de dependência"
+      },
+      "engines": {
+        "editor_label": "editor de rotulos"
+      },
+      "frigate": {
+        "camera_name": "Nome da câmera do Frigate (detectado automaticamente pela entidade)",
+        "client_id": "ID do cliente do Frigate (para >1 servidor Frigate)",
+        "labels": "rotulos",
+        "editor_label": "Opções do Frigate",
+        "url": "URL do servidor Frigate",
+        "zones": "zonas"
+      },
+      "go2rtc": {
+        "editor_label": "editor de rotulos",
+        "modes": {
+          "editor_label": "editor de rotulos",
+          "mse": "mse",
+          "webrtc": "webrtc",
+          "mp4": "mp4",
+          "mjpeg": "mjpeg"
+        },
+        "stream": "stream"
+      },
+      "hide": "esconder",
+      "icon": "Ícone para esta câmera (detectado automaticamente pela entidade)",
+      "image": {
+        "editor_label": "editor etiquetas",
+        "refresh_seconds": "refresh em segundos",
+        "url": "link"
+      },
+      "id": "ID exclusivo para esta câmera nesse cartão",
+      "live_provider": "Provedor de visualização ao vivo para esta câmera",
+      "live_provider_options": {
+        "editor_label": "editor de etiquetas"
+      },
+      "live_providers": {
+        "auto": "Automatico",
+        "jsmpeg": "JSMpeg",
+        "go2rtc": "go2rtc",
+        "ha": "ha",
+        "image": "imagem",
+        "webrtc-card": "Cartão WebRTC (de @AlexxIT)"
+      },
+      "motioneye": {
+        "editor_label": "directory_pattern",
```

Comment:
I don't think this is correct, in general it should never be the case that the value on the right is the same as the key on the left and I see several examples here where this is the case (e.g. `directory_pattern`, `file_pattern`), etc. The text on the right is displayed directly in the UI, so always needs to be a word or phrase that's easily consumable (i.e. nothing programmatic, no underscores, etc).

Reply:
ok i will fix that :)